07-routing-gossip.md

```diff
@@ -280,6 +280,13 @@ The following `address descriptor` types are defined:
          `[32:32_byte_ed25519_pubkey] || [2:checksum] || [1:version]`, where
          `checksum = sha3(".onion checksum" | pubkey || version)[:2]`.
 
+The following feature bits are currently assigned by this specification:
```

Comment:
nit: shouldn't feature flags be speced in Bolt 9 instead of scattered accross bolts?

Reply:
Good catch, I misread bolt 09 to be specific for `globalfeatures` and `localfeatures`. Fixup appended.